### ISSUE TITLE
C3M6 is way too hard

### ISSUE THREAD

**Issue by ghost** [freedoom/freedoom#318]
The 1st half of the map is good but after you encounter lots of "barons of hell" there is only a single medkit for the rest of the level. It makes it very hard and frustrating.

**Comment by Voros2** [comment]
Maybe more medkits would be good.

I just played it now, and it was pretty simple. You are given so much ammo, especially rockets. Sure the map requires lots of strafing though.

**Comment by FaTony** [comment]
I ended up with 10 hp after barons and kept dying to lost souls and other monsters who managed to hit me once.

**Comment by Voros2** [comment]
Why don't you record a demo of you playing the map? I'd like to see how you play.

**Comment by FaTony** [comment]
That would mean campaign from the first level, something I don't want to do.

**Comment by Voros2** [comment]
No it wouldn't. Just use the -warp command say "doom -skill 4 warp 3 6 -record e3m6" (this loads up doom.exe, sets the difficulty to 4 (Ultra-Violence) and warps to Episode 3 Map 6 and starts recording your game in the file "e3m6.lmp". To stop the recording, simply exit the game.

**Comment by FaTony** [comment]
I don't think I will be able to beat it without the weapons from previous level. This was on difficulty level 2, I forgot how it's called.

**Comment by Voros2** [comment]
Try. I just want to see how you play.
Although medkits should be in the map, as on skill 4, the group of barons encountered get replaced with a cyberdemon.

**Comment by FaTony** [comment]
I've already uninstalled the game because I've finished it.

**Comment by Voros2** [comment]
Then DL it again. Play in a higher skill level (2 is pretty weak, isn't it?)

**Comment by Proluiz** [comment]
i can try a gameplay too i'm very good at vanilla gameplay (since i play most complex doom, so normal doom seen a easy game now)

**Comment by Proluiz** [comment]
i'm having issues too, the cyberdemon fight is hard in prboom (where i tried to record) autoaim really f--- me off and the area is too small to avoid the spash dmg.

**Comment by Voros2** [comment]
On the contrary, the area is pretty big. But i guess this specific part of the map needs good skills to aviod getting hit too many times (the ratio of me winning to me dying/low health is basically 1:10).
Medkits should be added.

**Comment by Proluiz** [comment]
i did a run in zadronum (whitout jump and crouch) whit freelook on well i did the maping losing exactly 10hps only, i was hit by lost soul. maybe is the lack of free aim that kill me.

**Comment by JohnnyTheWolf** [comment]
I was playing Freedoom Phase 1 with a mod I made for myself, which removes monster respawning and the -fast attribute, but makes health and ammo less plentiful on Nightmare (Insanity). It worked well in the first two episodes, as they were rather easy, but then, I made it to the third episode and man, did it get hard - and by which I mean Plutonia hard. E3M6 is just impossible with my mod and it is still quite hard in unmodified Extreme Carnage.
